nexus/db-queries/src/db/datastore/silo.rs

```diff
@@ -28,6 +28,7 @@ use chrono::Utc;
 use diesel::prelude::*;
 use nexus_db_model::Certificate;
 use nexus_db_model::ServiceKind;
+use nexus_db_model::SiloQuotas;
```

Comment:
(as discussed on our call) It's down in `load_builtin_silo` but we need to insert the quotas here too!

Reply:
TODO for myself: create an issue for the bifurcation of silo creation